src/dune_rules/lib_info.ml

```diff
@@ -510,9 +515,12 @@ let for_dune_package t ~name ~ppx_runtime_deps ~requires ~foreign_objects
           | Some src_dir ->
             Path.source src_dir |> Path.to_absolute_filename |> Path.of_string))
   in
-  let native_archives =
-    Files (eval_native_archives_exn t ~modules:(Some modules))
+  let modules =
+    Option.map t.modules ~f:(fun modules ->
+        let install_dir = Obj_dir.dir obj_dir in
+        Modules.version_installed modules ~install_dir)
```

Comment:
Curious as to why you moved the call from `to_dune_lib`

Reply:
moved it back. artifact from a previous iteration